Mathlib/RingTheory/LaurentSeries.lean

```diff
@@ -380,12 +380,9 @@ theorem coe_X : ((X : RatFunc F) : F⸨X⸩) = single 1 1 := by
 
 theorem single_one_eq_pow {R : Type*} [Semiring R] (n : ℕ) :
     single (n : ℤ) (1 : R) = single (1 : ℤ) 1 ^ n := by
-  induction n with
-  | zero => simp
-  | succ n h_ind =>
-    rw [← Int.ofNat_add_one_out, pow_succ', ← h_ind, HahnSeries.single_mul_single, one_mul,
-      add_comm]
+  simp
 
+@[deprecated HahnSeries.inv_single (since := "2025-11-07")]
```

Comment:
I don't understand. This lemma seems to be about `LaurentSeries` and `HahnSeries.inv_single` is probably about `HahnSeries`? Or is the defeq abuse `LaurentSeries = HahnSeries Z` intentional? I would suppose that copying the API here is better than users needing to look in both `LaurentSeries` and `HahnSeries` to find the right lemma. 

Disclaimer: I have never used `HahnSeries` and the comments above are based on my experience with `PowerSeries/MvPowerSeries`.

Reply:
Okay there doesn't even seem to be a `LaurentSeries.single` so I guess this is fine.

Reply:
(or rather not fine but shouldn't block this PR)